tcms_tenants/templates/tcms_tenants/new.html

```diff
@@ -41,6 +42,9 @@
                                class="form-control" required
                                {% if form.schema_name.field.disabled %}disabled{% endif %}>
                         <span class="input-group-addon">.{{ tcms_tenants_domain }}</span>
+                        <span class="input-group-addon">
+                            {% attachment_form object%}
+                        </span>
```

Comment:
Does this work (or crash) when trying to create a new tenant vs. when editing an existing tenant? 

I think that you can't attach files to a non existing object so this section needs to be safeguarded by an if condition.